### PR TITLE
Update about.md and sagemaker/getting-started.md

### PR DESCRIPTION
**Pull request by ShubhamJagtap2000**
There was a glitch in setting headlines for the first 2 lines in the `docs` section, I have edited it and now they are visible as they were expected. 

Also, in the https://github.com/huggingface/hub-docs/blob/main/docs/sagemaker/getting-started.md 
if you see the YouTube video frame is not available as expected as the <iframe> is no longer working with Markdowns.

So, I have added the YouTube video link for the Sagemaker Workshop with a frame as expected.

## Before changes screenshot

image

## After Changes screenshot

image

Please review.

Thank you!

### CODE REVIEW

**Comment by julien-c** [comment]
Hmm, the video iframe does display for me on https://huggingface.co/docs/sagemaker/getting-started, not for you?

I agree with the header fixes though

File: tasks/src/image-classification/about.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
-## Use Cases
+## Use Cases
 Image classification models can be used when we are not interested in specific instances of objects with location information or their shape.
 
-### Keyword Classification
+### Keyword Classification
```
Comment (julien-c) [inline tasks/src/image-classification/about.md:4]: thanks for this fix!

**Comment by ShubhamJagtap2000** [comment]
> Hmm, the video iframe does display for me on https://huggingface.co/docs/sagemaker/getting-started, not for you?
> 
> I agree with the header fixes though

Oh, I was to tell about the in-GitHub documentation, @julien-c. 

In the docs, it was only the .md code being shown up, so I made it populate in GitHub MD file too.

Should we keep these changes in docs perspective or I should revert these changes and only keep header fixes? Will you be able to approve the PR?

Thank you.

**Comment by julien-c** [comment]
yes, please keep the header fixes only. Thanks!

**Pull request closed without merging**